src/hotspot/share/opto/superword.cpp

```diff
@@ -2359,472 +2052,6 @@ void SuperWord::schedule_reorder_memops(Node_List &memops_schedule) {
   }
 }
 
-//------------------------------output---------------------------
-// Convert packs into vector node operations
-// At this point, all correctness and profitability checks have passed.
-// We start the irreversible process of editing the C2 graph. Should
-// there be an unexpected situation (assert fails), then we can only
-// bail out of the compilation, as the graph has already been partially
-// modified. We bail out, and retry without SuperWord.
-bool SuperWord::output() {
```

Comment:
Note: this was a loop over all packs, and for each pack there is a big switch that determines how the pack is to be turned into a `VectorNode`.

Now, I split this method up into:
- `SuperWordVTransformBuilder::build_vtransform` -> `SuperWordVTransformBuilder::build_vector_vtnodes_for_packed_nodes` -> `SuperWordVTransformBuilder::make_vtnode_for_pack`.
  - Here we create the respective `VTransform`.
- `SuperWordVTransformBuilder::build_vtransform` -> `SuperWordVTransformBuilder::build_edges_for_vector_vtnodes`.
  - Here we ensure the `VTransform` nodes have the correct inputs.
- `VTransform...Node::apply`: create the C2 vector nodes.

This splitting up allows us to "model" the vectorized code without directly creating C2 nodes (i.e. applying). I have many future improvements planned that will be based on this `VTransform` model.